pulpcore/cli/file/repository.py

```diff
@@ -202,27 +202,44 @@ def remove(
     )
 
 
+def _load_json_from_option(
```

Comment:
You can turn this function into an option callback. Then you will just have the python objects in the command body.

https://click.palletsprojects.com/en/7.x/options/#callbacks-and-eager-options